### DISCUSSION TITLE
Let's allow for multiple ranges of dates to be disabled. I fix this ;)

### DISCUSSION THREAD

**Discussion by LeftCoder** [elreco/vue-tailwind-datepicker#48] [Ideas]
If I'm not mistaken current implementation allows for only one date range to be disabled.
Reffering to the docs: https://vue-tailwind-datepicker.com/props.html#disable-date
I wonder if we can do something to allow for wider selection of dates to be disabled?

**Example**
1. Disable all weekends in a year.
2. Disable specific periods of a year eg. holidays

Would be great to implement this feature and it could be inspired by flatpickr.
https://flatpickr.js.org/examples/#disabling-specific-dates

I'm willing to help you build the solution cus I badly need this for my production project.
What do you think?

**Comment by elreco** [comment]
Hey @LeftCoder 
First of all, thank you for using vue tailwind datepicker and taking the time to ask this question.

Of course, you can make a PR if you want! It would be very great. I will review it quickly 😀

**Reply by Kiansa** [reply]
Did you guys manage to implement this? I also need to disable multiple date ranges. my use case is for booking vacation rentals.

**Reply by LeftCoder** [reply]
Yes I can do this. Also need this for rental system btw ;)
